src/components/auction/Orderbook/index.tsx

```diff
@@ -59,8 +59,8 @@ export const OrderBook: React.FC<OrderbookProps> = (props) => {
 
   if (showChartsInverted(baseToken)) {
     for (const p of processedOrderbook) {
-      p.priceNumber = getInverse(p.price, NUMBER_OF_DIGITS_FOR_INVERSION)
-      p.priceFormatted = getInverse(p.price, NUMBER_OF_DIGITS_FOR_INVERSION).toString()
+      p.priceNumber = Number(getInverse(p.price.toString(), NUMBER_OF_DIGITS_FOR_INVERSION))
```

Comment:
This is the only place where a problem would arise, as Number(`-`) = NAN

But since   p.priceNumber = NAN is not causing problems, AFAIK, I would leave it that way. I am not sure how to better deal with it. Not showing the orderbook in such a case is not an option for me.